internal/repositories/vehicles.go

```diff
@@ -137,31 +137,31 @@ func (v *Repository) GetVehicles(ctx context.Context, first *int, after *string,
 
 	if first != nil {
 		if last != nil {
-			return nil, errors.New("Pass `first` or `last`, but not both.")
+			return nil, errors.New("pass `first` or `last`, but not both")
 		}
 		if *first < 0 {
-			return nil, errors.New("The value for `first` cannot be negative.")
+			return nil, errors.New("the value for `first` cannot be negative")
 		}
 		if *first > maxPageSize {
-			return nil, fmt.Errorf("The value %d for `first` exceeds the limit %d.", *last, maxPageSize)
+			return nil, fmt.Errorf("the value %d for `first` exceeds the limit %d", *last, maxPageSize)
 		}
 		limit = *first
 	} else {
 		if last == nil {
-			return nil, errors.New("Provide `first` or `last`.")
+			return nil, errors.New("provide `first` or `last`")
 		}
 		if *last < 0 {
-			return nil, errors.New("The value for `last` cannot be negative.")
+			return nil, errors.New("the value for `last` cannot be negative")
 		}
 		if *last > maxPageSize {
-			return nil, fmt.Errorf("The value %d for `last` exceeds the limit %d.", *last, maxPageSize)
+			return nil, fmt.Errorf("the value %d for `last` exceeds the limit %d", *last, maxPageSize)
 		}
 		limit = *last
 	}
 
+	var err error
 	var totalCount int64
 	var queryMods []qm.QueryMod
-
 	if filterBy != nil && filterBy.Privileged != nil {
```

Comment:
What if someone places both filters on?

Reply:
I think that case is already accounted for in the privileges query because they inherently have privileges on vehicles they own, this is the portion of the privileges query that seems to accomplish that:

```
	qm.Expr(
		models.VehicleWhere.OwnerAddress.EQ(addr.Bytes()),
		qm.Or2(
			qm.Expr(
				models.PrivilegeWhere.UserAddress.EQ(addr.Bytes()),
				models.PrivilegeWhere.ExpiresAt.GTE(time.Now()),
			),
		),
	),
```

Reply:
Suppose we have

```
Vehicle | Owner | Privileged Users
--------+-------+-----------------
1       | A     | B
2       | B     | C
```

If I ask for vehicles which `A` owns and to which `B` has privileges, I'm going to get _both_ of these. I guess the fundamental question is whether you think this is an OR or an AND. I think it's an AND.